tests/js/setup.js

```diff
@@ -100,24 +101,19 @@ jest.mock('echarts-for-react/lib/core', () => {
 
 jest.mock('@sentry/browser', () => {
```

Comment:
@zivl maybe a pattern we should document

Reply:
yeah this actually makes sense!
I've opened an issue at our end to put in the docs as well for suggested example...
https://github.com/wix/sentry-testkit/issues/44
maybe even predefined-ready-to-use-jest-mock? not sure but we'll check out that path